src/proto/streams/state.rs

```diff
@@ -317,6 +324,14 @@ impl State {
         }
     }
 
+    // /// Returns true if the stream was closed by sending an EOS.
+    // pub fn is_eos(&self) -> bool {
```

Comment:
Oops, I meant to mention this in the first review, would it be possible to remove this commented out code?

Reply:
Oh, yeah, missed that when committing. Sorry.